index.html

```diff
@@ -5281,42 +5396,44 @@ <h3>Identifier-Based Correlation</h3>
         <h3>Signature-Based Correlation</h3>
 
         <p>
-The contents of a [=credential=] are secured using a securing mechanism.
-Values used to represent the securing mechanism
-create a greater risk of correlation when the same values are used across more
-than one session or domain and the value does not change.
+The contents of a [=verifiable credential=] are secured using a [=securing
+mechanism=]. Values used to represent the securing mechanism create a greater
+risk of correlation when the same values are used across more than one session
+or domain and the value does not change. Examples of these sorts of values
+include:
         </p>
 
+        <ul>
+          <li>
+the binary value of the digital signature,
+          </li>
+          <li>
+timestamp information associated with when the digital signature was created,
+and
+          </li>
+          <li>
+cryptographic material, such as a public key identifier, associated with the
+digital signature.
+          </li>
+        </ul>
+
         <p>
 If strong anti-correlation properties are required, it is advised that
-signature values and metadata are regenerated each time using technologies like
-third-party pairwise signatures, zero-knowledge proofs, or group signatures.
+[=issuers=] produce [=verifiable credentials=] where the
+signature values and metadata can be regenerated for each
+[=verifiable presentation=] using technologies that support unlinkable
+disclosure, such as the [[[?VC-DI-BBS]]] specification. It is advised that, if
+possible, [=verifiers=] prefer [=verifiable presentations=] that use this
+technology in order to enhance the privacy for [=holders=] and [=subjects=].
```

Comment:
This paragraph seems like it's crafted to be an advertisement for a DI spec.  Please revert this proposed change.  The existing text already did the job.

Reply:
More specific guidance on unlinkability and which roles in the ecosystem are responsible for unlinkability was requested by @jyasskin. If VC-JOSE-COSE supported unlinkable disclosure, we could refer to it as well. Speaking to unlinkability, but then not providing an example of it, would not achieve what the issue commenter was requesting to be addressed.